tools/shared.py

```diff
@@ -1650,9 +1650,9 @@ class ret:
           ret.undefs.append(symbol)
         elif status == 'C':
           ret.commons.append(symbol)
-        elif status == status.upper(): # all other uppercase statuses ('T', etc.) are normally defined symbols
```

Comment:
The code is fine as it is. But we should just watch the bots more carefully just to be sure. Let's merge this.